addon/templates/components/polaris-radio-button.hbs

```diff
@@ -1,27 +1,27 @@
-{{#polaris-choice
-  inputId=_id
-  label=label
-  labelHidden=labelHidden
-  disabled=disabled
-  helpText=helpText
-}}
-  <span data-test-radio-button class="Polaris-RadioButton">
+<PolarisChoice
+  @inputId={{this._id}}
+  @label={{@label}}
+  @labelHidden={{@labelHidden}}
+  @disabled={{@disabled}}
+  @helpText={{@helpText}}
+>
+  <span data-test-radio-button="true" class="Polaris-RadioButton">
     <input
       data-test-radio-button-input
       type="radio"
       class="Polaris-RadioButton__Input"
-      id={{_id}}
-      name={{name}}
-      value={{value}}
-      checked={{checked}}
-      disabled={{disabled}}
-      aria-describedby={{describedBy}}
-      onchange={{action onChange value="target.value"}}
-      onfocus={{action onFocus}}
-      onblur={{action onBlur}}
+      id={{this._id}}
+      name={{@name}}
+      value={{@value}}
+      checked={{@checked}}
+      disabled={{@disabled}}
+      aria-describedby={{this.describedBy}}
+      onchange={{action (or @onChange this.onChange) value="target.value"}}
```

Comment:
Left the `action` helper here because the `fn` wasn't working as expected when passing `value="target.value"` 🤷‍♂

Reply:
@tomnez can you try 
`<input ...... {{on "change" (or @onChange this.onChange)}} ....>`

Reply:
This still passes the `event` as the first (and only) argument now rather than the expected value 🤷‍♂

Reply:
not really a problem, we can just handle in the local action right? as the example [here](https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/)

<img width="769" alt="Screen Shot 2020-02-17 at 4 09 59 PM" src="https://user-images.githubusercontent.com/160955/74669925-093f9300-51a0-11ea-9063-ffcd5081f2d5.png">

Reply:
Yeah we can do that 👍